index.html

```diff
@@ -0,0 +1,2320 @@
+                      src="assets/img/donald-trump-melania-trump.webp"
+                      class="card-img rounded-0"
+                      alt="george-floyd"
+                  /></a>
+                </div>
+                <div class="col-9">
+                  <div class="card-body p-0 pl-2">
+                    <h6 class="font-weight-bold my-link-style">
+                      <a class="text-reset text-decoration-none" href="#">
+                        George Floyd's Friend Says It Was 'Heartbreaking' to
+                        Find Out How He Died</a
+                      >
+                    </h6>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div
+              class="card px-1 py-2 border-left-0 border-right-0 border-bottom-0 border-top-0 rounded-0"
+            >
+              <div class="row no-gutters">
+                <div class="col-3 rounded-0">
+                  <a href="#">
+                    <img
+                      src="assets/img/donald-trump-melania-trump.webp"
+                      class="card-img rounded-0"
+                      alt="george-floyd"
+                  /></a>
+                </div>
+                <div class="col-9">
+                  <div class="card-body p-0 pl-2">
+                    <h6 class="font-weight-bold my-link-style">
+                      <a class="text-reset text-decoration-none" href="#">
+                        George Floyd's Friend Says It Was 'Heartbreaking' to
+                        Find Out How He Died</a
+                      >
+                    </h6>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="row ml-3">
+          <div class="col-lg-8 bg-light row be-fo">
+            <div class="col-sm-12 col-md-5 img-cont pr-0 position-relative">
+              <div class="position-absolute subscribe-img-2">
+                <img
+                  src="assets/img/may-29-june-06-2020-cover.jpg"
+                  alt="may-29-june-06-2020-cover"
+                />
+              </div>
+              <div class="position-absolute subscribe-img-1">
+                <img
+                  src="assets/img/may-22-2020-cover.jpg"
+                  alt="may-22-2020-cover"
+                />
+              </div>
+            </div>
+            <div class="col-sm-12 pl-0 col-md-7 mt-5 pr-2 pt-2">
+              <h5 class="font-weight-bold">
+                In-depth Stories You Don't See Elsewhere About Topics You Want
+                to Read!
+              </h5>
+              <p>
+                All Access + Weekly Delivery. Choose A Membership That's Perfect
+                for You!
+              </p>
+              <button
+                class="btn red-back px-2 py-1 text-uppercase text-light font-weight-bold"
+              >
+                Subscribe >
+              </button>
+            </div>
```

Comment:
**[REQUIRED]** kindly fix the text in this block. It is below the image and hence difficult to read on mobile and medium view
![snapshot-1591175528832](https://user-images.githubusercontent.com/32361963/83619147-1b4d1200-a594-11ea-9efc-b32612e79811.jpg)